docs/user-guide/get-started/settings-definition.md

```diff
@@ -21,6 +21,7 @@ The following section lists the configurable parameters. If the user does not sp
 |[`separation_parameter`](#separation_parameter) | `0.5` | Step size for the in-out separation |
 |[`relaxed_optimality_gap`](#relaxed_optimality_gap) | `1e-5` | Threshold to switch from relaxed to integer master |
 |[`batch_size`](#batch_size) | `0` | Number of subproblems per batch |
+|[`expert_logs`](#expert_logs) | `false` | display pure optimization data (/!\ the corresponding option(`EXPERT_LOGS`) for benders is `true` by default ) |
```

Comment:
We need to rename/reorganize solver related log options to make it clear for the user. We have several logs :
- Operational logs, always stored in a file, printed in the terminal if `expert_logs` deactivated
- Benders solver logs, aka `expert_logs`
- Xpress solver logs i.e. lower level logs that are the logs of the linear solver used within Benders

We could either :
- Use the `log_level` settings with level 0 (operational logs), 1 (Benders logs), 2 (Benders + Xpress logs)
- Define clear names for each log that take boolean value defining whether this log has to be saved such as :
  - `benders_log` instead of `expert_logs` (more meaningful)
  - `linear_solver_logs` for the Xpress logs

Reply:
Besides, `log_level` (Xpress logs) and `expert_logs` do not have the same behavior.

If `log_level = 0`, Xpress logs are not printed to the file, whereas it is the case for Benders logs. We need to uniformize the behaviors.

And we specify `log_level` with a numerical value and `expert_logs` with a boolean.

Maybe the simpler is to handle all this with the `log_level` option with the value indicating the complexity of logs to save, then we only have one parameter to handle

Reply:
You have just given a specification 👍